framework/source/class/qx/ui/splitpane/Pane.js

```diff
@@ -299,11 +313,11 @@ qx.Class.define("qx.ui.splitpane.Pane",
         }
         var left = bounds && bounds.left;
 
-        if (width) {
+        if (width || !this.getSplitterVisible()) {
           if (isNaN(left)) {
             left = qx.bom.element.Location.getPosition(splitterElem).left;
           }
-          this.__blocker.setWidth(offset, width);
+          this.__blocker.setWidth(offset, width || 6);
```

Comment:
@oetiker can you be more specific? I would think in the exact opposite direction (if I understand you correctly): make the 6px width part of the theme so that it can be overridden.

Reply:
the size of the blocker is calculated from the width of the splitter plus the offset ... so in this setup, the offset will come from the theme while the width might be overridden by "6" if it happens to be 0 which seems odd ... and also this change goes into effect regardless of the setting of `splitterVisible`

Reply:
> the offset will come from the theme

But the offset is not a themeable property, it is a setting of the Pane. The width (or height) could be 0, coming from the theme. I think the original implementation is thought in a way that it could be possible to disable the resizing of the panes, setting them to 0 like that, so I wanted to respect that, unless the splitterVisible property is set to false, then understanding that the user wants to hide the splitter, but keep using the resize feature.